src/Storageless/Http/SessionMiddleware.php

```diff
@@ -115,33 +99,39 @@ public static function fromSymmetricKeyDefaults(string $symmetricKey, int $expir
      * This constructor simplifies instantiation when using HTTPS (REQUIRED!) and asymmetric key encryption
      * based on RSA keys
      */
-    public static function fromAsymmetricKeyDefaults(
-        string $privateRsaKey,
-        string $publicRsaKey,
+    public static function fromRsaAsymmetricKeyDefaults(
```

Comment:
Rsa are not the only asymmetric signing available, there are also EcDSA and EdDSA, the latter the most secure and easy to use.